identity/README.md

```diff
@@ -261,6 +261,34 @@ message Capability {
     Type type = 1;
   }
 
+  // VolumeGroup contains the features of the volumegroup operation
+  // that the CSI-driver supports.
+  message VolumeGroup {
+    // Type describes a CSI Service that CSI-drivers can support.
+    enum Type {
+      // UNKNOWN indicates that the CSI-driver does not support the
+      // VolumeGroup operation in the current mode. The CSI-Addons CO
+      // plugin will most likely ignore this node for the
+      // VolumeGroup operation.
+      UNKNOWN = 0;
+
+      // VOLUME_GROUP indicates that the CSI-driver provides RPCs for a
+      // VolumeGroup operation.
+      // The presence of this capability determines whether the CSI-Addons CO
+      // plugin can invoke RPCs that require access to the storage system,
+      // similar to the CSI Controller (provisioner).
+      VOLUME_GROUP = 1;
+      // LIMIT_PVC_TO_ONE_VG indicates that the CSI-driver does not support
+      // that one volume will be in multiple VGs, so it can be in only one VG.
+      LIMIT_PVC_TO_ONE_VG = 2;
```

Comment:
try not to use PVC, PV or other Kubernetes terminology. Ideally CSI-Addons is container orchistrator agnostic. The comments use "volume" correctly, consider using that in the enum too.

Reply:
done